dev/CheckBox/TestUI/CornerRadiusPage.xaml

```diff
@@ -0,0 +1,544 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
```

Comment:
Should this page be in some other folder? CommonStyles?

Reply:
That's exactly where it's supposed to be, and where I thought I had it. Weird, how'd it get in the Checkbox folder? I'll move it...

Reply:
I just noticed you've moved it to CommonStyles folder in your other branch user/cglein/migrateCoreControlStyles.  So I guess it's fine to merge this in and let the other PR fix it.